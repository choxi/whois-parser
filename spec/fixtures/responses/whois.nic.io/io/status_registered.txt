Domain Name: REDIS.IO
Registry Domain ID: D503300000040423931-LRMS
Registrar WHOIS Server: whois.name.com
Registrar URL: www.name.com
Updated Date: 2018-04-13T20:45:25Z
Creation Date: 2010-05-28T22:09:44Z
Registry Expiry Date: 2019-05-28T22:09:44Z
Registrar Registration Expiration Date:
Registrar: Name.com LLC
Registrar IANA ID: 625
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Registrant Organization: Domain Protection Services, Inc.
Registrant State/Province: CO
Registrant Country: US
Name Server: NS2KQZ.NAME.COM
Name Server: NS1DJS.NAME.COM
Name Server: NS4BHT.NAME.COM
Name Server: NS3CKL.NAME.COM
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form is https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2018-08-08T22:49:18Z <<<
