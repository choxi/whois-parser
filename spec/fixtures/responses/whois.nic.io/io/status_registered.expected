#disclaimer
  %s %ERROR{AttributeNotSupported}

#domain
  %s == "redis.io"

#domain_id
  %s %ERROR{AttributeNotSupported}

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %ERROR{AttributeNotSupported}

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-05-28 22:09:44 UTC}

#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}

#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "NS2KQZ.NAME.COM"
  %s[1] %CLASS{nameserver}
  %s[1].name == "NS1DJS.NAME.COM"
  %s[2] %CLASS{nameserver}
  %s[2].name == "NS4BHT.NAME.COM"
  %s[3] %CLASS{nameserver}
  %s[3].name == "NS3CKL.NAME.COM"
